### DISCUSSION TITLE
instructions to running plausible from source?

### DISCUSSION THREAD

**Discussion by LukeWood** [plausible/analytics#4866] [Self-Hosted Support]
Hey - hows it going?  I love plausible.  Coincidentally, I also love elixir!  I'd love to self-host plausible CE but from source rather than the docker image.  Are there any instructions on how to get the app up and running from source?  Any tips would be greatly appreciated!

**Comment by ruslandoga** [comment]
👋 @LukeWood 

Dockerfile is a good starting point. You can ask ChatGPT or some other similar tool to translate it for your system :)

**Comment by LukeWood** [comment]
I also found the contributing guide which helped a lot !  Thanks
